shared/notifications/index.js

```diff
@@ -24,7 +24,7 @@ export const UI_NOTIFICATIONS = {
   },
   5: {
     id: 5,
-    date: '2021-05-18',
+    date: '2021-06-09',
```

Comment:
Since notification order is sorted by date by the `getSortedNotificationsToShow` selector, it made sense to just to update the date of the Recovery phrase entry rather than re-order any other static structures